### PR TITLE
linaro-forge: added 24.0.4 version

### PR DESCRIPTION
**Pull request by louspe-linaro**

### CODE REVIEW

**Review by kenche-linaro — APPROVED** [review]
LGTM

**Comment by wdconinc** [comment]
For information, I'm noting here that the package `ddt` seems to be a duplicate of `linaro-forge` (same hash). In a separate PR it would make sense to deprecate and later remove `ddt` since this seems to be the more complete and better maintained package.

**Comment by louspe-linaro** [comment]
> For information, I'm noting here that the package `ddt` seems to be a duplicate of `linaro-forge` (same hash). In a separate PR it would make sense to deprecate and later remove `ddt` since this seems to be the more complete and better maintained package.

Thanks for creating the pull request to remove `ddt` @wdconinc

File: var/spack/repos/builtin/packages/linaro-forge/package.py [hunk 1]
```diff
@@ -23,6 +23,9 @@ class LinaroForge(Package):
     maintainers("kenche-linaro")
 
     if platform.machine() in ["aarch64", "arm64"]:
```
Comment (wdconinc) [inline var/spack/repos/builtin/packages/linaro-forge/package.py]: Verified new hash for aarch64 and x86_64.

Unable to download from url https://downloads.linaroforge.com/24.0.4/linaro-forge-24.0.4-linux-arm64.tar for `platform.machine() == "arm64"`.  I don't think `platform.machine()` can have that value, though. `spack arch` doesn't list it.

Comment (louspe-linaro) [reply var/spack/repos/builtin/packages/linaro-forge/package.py]: Modified the pull request to remove `arm64` and updated commit message.

**Review by wdconinc — APPROVED** [review]

**Pull request merged**